README.md

```diff
@@ -155,7 +177,8 @@ rush remove -p <package-name>     # Remove a dependency
 - **[AGENTS.md](AGENTS.md)** - Guidelines for AI coding agents working with this codebase
 - **[docs/MONOREPO.md](docs/MONOREPO.md)** - Detailed monorepo structure and usage guide
 - **[docs/ADDING_PACKAGES.md](docs/ADDING_PACKAGES.md)** - Step-by-step guide for adding new packages
-- **[docs/SUBTREE.md](docs/SUBTREE.md)** - Git subtree management guide for OpenFrontIO
+- **[docs/WORKTREE_WORKFLOW.md](docs/WORKTREE_WORKFLOW.md)** - Git worktree workflow guide for OpenFrontIO
+- **[docs/SUBTREE.md](docs/SUBTREE.md)** - Git subtree management guide (legacy, for reference)
```

Comment:
Go ahead and kill the Subtree documentation, and scrub any extraneous information about subtrees or paths to the subtree documentation file from the rest of the documentation.